### PR TITLE
View tags

### LINKED ISSUE #8: Images by Tag
#### As a user I want to view all images associated with a tag.

__Story__:

Now that we have added the ability for images to be associated with tags, let
us provide the ability for users to filter images by a given tag. Nana, who
loves fast planes, will be delighted to see only images labeled with SR-71.

__Acceptance criteria__:
- [x] When I click on a tag, I see a filtered list of only the images that have
  the tag I clicked on.
- [x] The tag name, not its ID, is used to perform the filtering.

__Discussion topics__:
- [ ] Where in the code should we implement this functionality?
- [ ] What happens if a user tries to manually filter by a nonexistent tag?

__Dependencies__:
- Image Tags

### PR DESCRIPTION
**Pull request by lelandmiller**
Closes #8

### CODE REVIEW

**Comment by lelandmiller** [comment]
[[Ropes] View tags](https://trello.com/c/Md1dBBbm/1084-ropes-view-tags)

File: app/controllers/tags_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class TagsController < ApplicationController
+  def show
+    @tag = params[:tag]
+    @images = Image.order(created_at: :desc).tagged_with(@tag)
```
Comment (zldavis) [inline app/controllers/tags_controller.rb]: really minor, but generally in terms of code style the `order` clause would be after any scopes/conditions in the query.

**Comment by lelandmiller** [comment]
That makes sense. I moved the action to the images controller, let me know if this looks closer to an idiomatic way of doing things.

Interestingly, when doing the flow testing followup stories I noticed that the flow tests were written from the perspective of the landing page being an index for images as well. The way the ropes issues are laid out we had a landing page before images, which is why I ended up here. Although I guess it makes sense that eventually we might want other non-image concerns in that controller as well. Does it make sense to have the landing page separate as well?

**Comment by lelandmiller** [comment]
As a note, I also made it so we check that tags show on the landing page in here as well (though I do not check that link correctly).

**Pull request merged**